blackbox/docs/sql/information_schema.txt

```diff
@@ -27,63 +27,41 @@ settings like the ``number of shards`` or ``number of replicas``.
+| ``number_of_replicas``           | The number of replicas the table currently has                             | ``Integer`` |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``number_of_shards``             | The number of shards the table is currently distributed across.            | ``Integer`` |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``partitioned_by``               | The column used to partition the table                                     | ``String``  |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``reference_generation``         | Not implemented in CrateDB                                                 |             |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``routing_hash_function``        | The hash function used for internal routing                                | ``String``  |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``self_referencing_column_name`` | Not implemented in CrateDB                                                 |             |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``settings``                     | :ref:`conf_table_settings`                                                 | ``Object``  |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``table_catalog``                | The name of the database (always ``CrateDB``)                              | ``String``  |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``table_name``                   | The name of the table                                                      | ``String``  |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``table_schema``                 | The name of the schema that the table belongs to                           | ``String``  |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``table_type``                   | The type of the table (always ``BASE TABLE``)                              | ``String``  |
++----------------------------------+----------------------------------------------------------------------------+-------------+
+| ``version``                      | Version information about when the table was created/upgraded              | ``Object``  |
```

Comment:
version of what? version of crate? I think this needs rewording

also, the "/" should be replaced with an "and" or "or", and possibly the sentence restructured to make it reader better without the slash